### PR TITLE
Minor corrections to deployment guide

### PR DESCRIPTION
**Pull request by ironcladlou**

### CODE REVIEW

**Comment by ironcladlou** [comment]
ptal @pmorie @nhr

**Comment by nhr** [comment]
LGTM; pinging @adellape or @CowboysFan for review / merge.

**Comment by pmorie** [comment]
LGTM

File: using_openshift/deployments.adoc [hunk 1]
```diff
@@ -138,13 +136,13 @@ Additionally, the following environment variables are provided by OpenShift to t
...
-|The name of the `replicationController` representing the new `deployment`
+|The name of the new `deployment` (a `replicationController`)
 
 .^|`OPENSHIFT_DEPLOYMENT_NAMESPACE`
-|The namespace of the `replicationController` representing the new `deployment`
+|The namespace of the new `deployment`
 |===
 
-The replica count of the `replicationController` for the new deployment will be 0 initially. The responsibility of the `strategy` is to make the new `deployment` live using whatever logic best serves the needs of the user.
+The replica count the new `deployment` will be 0 initially. The responsibility of the `strategy` is to make the new `deployment` live using whatever logic best serves the needs of the user.
```
Comment (adellape) [inline using_openshift/deployments.adoc]: s/count the/count of the/

Comment (ironcladlou) [inline using_openshift/deployments.adoc]: Good catch; fixed.

**Pull request merged**